src/DotNetWorker/Hosting/FunctionsApplication.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Microsoft.Extensions.Hosting
```

Comment:
Asking to make sure it's an intentional choice: do we want this to be in `Microsoft.Extensions.Hosting`?

Reply:
Ahh good point. AspNet uses `Microsoft.AspNetCore.Builder` as their namespace -- we should probably mirror that.